### ISSUE TITLE
Rocket chat is not working on Edge

### ISSUE THREAD

**Issue by barak-haviv** [RocketChat/Rocket.Chat#13110]
### Description:

On Microsoft Edge, it is not getting new messages only after refresh.
I.e WebSockets not working

### Steps to reproduce:

1. https://open.rocket.chat/ on Edge
2. open rocket chat on any other browser
3. write a new text message
4. message isn't received

### Server Setup Information:

- Version of Rocket.Chat Server: 0.72.3 / 0.73.1 - test on both versions
- Operating System: ubuntu
- Deployment Method: snap also tar
- Number of Running Instances: 1
- DB Replicaset Oplog: 
- NodeJS Version: 8.11.3

Duplicate of https://github.com/RocketChat/Rocket.Chat/issues/13024

**Comment by robertwessen** [comment]
Same symptoms here.

**Comment by barak-haviv** [comment]
I check the versions it is working on version 0.71.2
And as I said before not working 0.72.3
Looking at the diff not make so much sense.
I hope someone can take a look and give us any input.

https://github.com/RocketChat/Rocket.Chat/compare/0.72.3...release-0.71.2

**Comment by tassoevan** [comment]
It is well known that upgrading Meteor caused this, because Meteor 1.8 introduced a mechanism to provide smaller javascript bundles for modern browsers that don't need some polyfills. I'm not sure which ones, but I've seen that some Edge versions handle things wrongly. @barak-haviv @robertwessen Could you add this info, Edge version?

**Comment by barak-haviv** [comment]
Microsoft Edge 44.17763.1.0
Microsoft EdgeHTML 18.17763

**Comment by robertwessen** [comment]
44.17763 here as well.

**Comment by robertwessen** [comment]
Is this as simple as adding additional User-Agents and feeding them the "legacy" bundle? Or does it require using the legacy bundle for all browsers? 

I applaud any efforts to cut down on unnecessary polyfills and taking advantage of meteor's logic here, but if it's broken, it's broken. I know Edge is a small portion of overall traffic, but its still large enough to warrant an official minor point release fix IMO.

(if it is just a meteor config change when building RC, could you point us to any temporary config change/workaround we can do locally while this is being addresses?)

**Comment by serviceman** [comment]
Still having the same issues in Edge Browser, I am getting dozens of reports a day from my users about this issue. Edge browser is the default browser on all Windows machines and I get a lot of users that use Edge browser, are there any updates on this issue? I am using the latest version 1.0.3 with the same issues. Also, Edge browser also causes many, many requests when opening a private chat, causing the "too many connections" error. If you have too many users attempting to use Edge browser, this could cause a mini DOSS attack on the server. For security and stability issues, I feel that this should get more attention.

**Comment by ggazzo** [comment]
@barak-haviv I don't think I'm able to reproduce this error, cloud you please test again? thanks

**Comment by MartinSchoeler** [comment]
@barak-haviv I've tested using Microsoft Edge 44.18362.387.0 on a windows 10 machine and it seems to be working (using the steps you provided on the issue description). I'll be closing this for now, if you are still facing this issue, please open a new issue with updated details and tell us if you experience this on open server or only on your own. Thanks!
